drivers/hmis/app/graphql/mutations/refresh_external_submissions.rb

```diff
@@ -0,0 +1,20 @@
+#  Copyright 2016 - 2024 Green River Data Analysis, LLC
+#
+#  License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+#
+
+module Mutations
```

Comment:
Should this indeed be a mutation (because it is, sorta kinda, mutating data on the backend) or is it more like a query?

Reply:
Yeah I think mutation since it's not returning any data. It is modifying data on the server by queuing up the fetch job.